### PR TITLE
Style to form

### PR DESCRIPTION
**Pull request by leigh-mil**
## Description
Initial styling updates to the TO Form. There is still some fine tuning that needs to be done, like the LOA fields.

## Pivotal
https://www.pivotaltracker.com/story/show/166330678

## Screenshots
Screen Shot 2019-06-12 at 2 15 37 PM
Screen Shot 2019-06-12 at 2 15 48 PM
Screen Shot 2019-06-12 at 2 15 56 PM

### CODE REVIEW

File: templates/components/totals_box.html [hunk 1]
```diff
@@ -1,14 +1,19 @@
-{% macro TotalsBox(task_order) -%}
+{% macro TotalsBox(task_order=None, obligated_funds=0, contract_amount=0) -%}
```
Comment (montana-mil) [inline templates/components/totals_box.html]: why this change? seems like extra lines of code for the same thing, no?

File: atst/routes/task_orders/new.py [hunk 2]
```diff
@@ -8,11 +8,12 @@
 from atst.utils.flash import formatted_flash as flash
 
 
-def render_task_orders_edit(portfolio_id, task_order_id=None, form=None):
+def render_task_orders_edit(portfolio_id=None, task_order_id=None, form=None):
```
Comment (dandds) [inline atst/routes/task_orders/new.py]: Do we need `portfolio_id` as an arg at all, now?

File: atst/routes/task_orders/new.py [hunk 3]
```diff
@@ -28,18 +29,16 @@ def render_task_orders_edit(portfolio_id, task_order_id=None, form=None):
 
 
 @task_orders_bp.route("/portfolios/<portfolio_id>/task_orders/new")
-@task_orders_bp.route("/portfolios/<portfolio_id>/task_orders/<task_order_id>/edit")
+@task_orders_bp.route("/task_orders/<task_order_id>/edit")
 @user_can(Permissions.CREATE_TASK_ORDER, message="view new task order form")
-def edit(portfolio_id, task_order_id=None):
+def edit(portfolio_id=None, task_order_id=None):
```
Comment (dandds) [inline atst/routes/task_orders/new.py]: We can just drop the `portfolio_id` kwarg from the route functions.
